### ISSUE TITLE
Blocked by Google

### ISSUE THREAD

**Issue by itguy64** [jberkel/sms-backup-plus#1064]
Before you report your issue, please consider perusing the [https://github.com/jberkel/sms-backup-plus#faq](FAQ) and searching the issues page to see if it has already been reported.

### Expected behaviour
log in to GMail

### Actual behaviour
Blocked by Google

### Steps to reproduce the behaviour
Try to connect to GMail account

### Please specify the following:

* Android version
* Phone model / brand
* SMS Backup+ version installed
* Messaging app

**Comment by kurahaupo** [comment]
This is a duplicate of issue #972; please see the instructions there for setting up IMAP access.
